src/index.ts

```diff
@@ -0,0 +1,350 @@
+import https from "https";
+import readline from "readline";
+import Enquirer from "enquirer";
+import getStdin from "get-stdin";
+import boxen, { BorderStyle } from "boxen";
+import chalk from "chalk";
+import cliCursor from "cli-cursor";
```

Comment:
Codacy found an issue: [No default export found in imported module "cli-cursor".](https://app.codacy.com/gh/P2P-Develop/TopDomainChecker/pullRequest?prid=6364165)